gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,91 +1,151 @@
+The following table summarizes the differences between Connected Mode and Local Mode.
 
-* Provides resilient service-to-service communication using zero-trust policies to handle traffic control, fault tolerance and load balancing
-* Extends MuleSoft API Management capabilities to any non-Mule service, with policies, security and analytics all applied directly from Anypoint Platform
-* Amplifies reuse of services with discovery of non-Mule services within Exchange, available for use in future projects
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| | Connected Mode | Local Mode
 
-For more information, see xref:service-mesh::getting-started-service-mesh.adoc[Get Started with Service Mesh].
+| *Use Case*
+| Choose Connected Mode for centralized management, observability, and security. Flex Gateway in Connected mode connects to the control plane.
+| Choose Local Mode to operate independently of the control plane in a mostly disconnected manner. Manage Flex Gateway in Local Mode with locally stored declarative configuration files. Use Local Mode to build CI/CD pipelines.
 
-// Anypoint Flex Gateway is an ultrafast API gateway designed to manage and secure APIs running anywhere. Built to seamlessly integrate with DevOps and CI/CD workflows, the Anypoint Flex Gateway delivers the performance required for the most demanding applications while providing enterprise security and manageability across any environment.
+| *Policy Application*
+| Use API Manager to apply policies in Connected Mode.
+| Apply policies in Local Mode via local declarative configuration files.
 
-// xref:flex-gateway-getting-started.adoc[Get Started with Flex Gateway]
+| *Air-Gapped?*
+| No. Flex Gateway is connected to the control plane.
+| No. Flex Gateway is managed locally but connected to the control plane for registration and usage metrics.
 
-// == Architecture
+|===
 
-// The following diagram describes overall Flex Gateway architecture, for both Local and Connected modes. Connected Mode includes the Anypoint Platform component. Local Mode is mostly disconnected from Anypoint Platform.
+== Installation Options
 
-// image::gateway-architecture-diagram.png[width=85%]
+You can install Flex Gateway in a variety of ways:
 
-// Flex Gateway consists of:
+* A standalone single runtime or replica on various Linux environments including Ubuntu, Debian, Amazon Linux, RHEL, and CentOS.
+
+* A standalone in a Docker container.
+
+* A sidecar to a backend application in a Docker container, thereby protecting a single backend application.
+
+* An Ingress controller in Kubernetes, for highly-available, high-performance use cases.
+
+In general, if the use case involves protecting a few APIs and the number of transactions are low, with a minimal impetus on high-availability or robustness, choose to install as a standalone gateway. Otherwise, choose to install as an Ingress controller in Kubernetes.
```

Comment:
The first sentence is a mouthful and a little hard to scan. 
What about something like:

Use standalone gateway when...
Use Ingress controller for...

Subject verb-agreement: number is

Reply:
Fixed